e2e-tests/arbiter/run

```diff
@@ -59,14 +64,16 @@ check_cr_config() {
 	compare_mongo_cmd "find" "myApp:myPass@$(get_service_ip $cluster-0)" "" ":27017"
 	compare_mongo_cmd "find" "myApp:myPass@$(get_service_ip $cluster-1)" "" ":27017"
 
-	# delete cluster
+	desc "delete PSMDB cluster $cluster"
```

Comment:
Let's use single quote or double quote everywhere wih `desc`. If we want to use variables why not just use double quote everywhere.

Reply:
Because it is bash best practices, you should not use  double quote  if it is unneeded